cosmic-settings/src/pages/desktop/appearance.rs

```diff
@@ -149,6 +154,17 @@ impl
                 && c != theme.palette.accent_yellow
         });
 
+        let panel_config_helper = CosmicPanelConfig::cosmic_config("Panel").ok();
```

Comment:
I think it would be ok to get the panel and dock configs in the update_panel_radii method. Is there a that they are added to the Page?

Reply:
No reason, fixed!